scripts/jenkins/ardana/ansible/group_vars/deployer_virt.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+cobbler_requires:
+  - name: "rhel7.iso"
+    url: "http://10.84.144.252/compute_iso/centos73.iso"
```

Comment:
that doesnt exist, there is only a centos75.iso (and I guess we should do testing against centos 7.5 anyway)

Reply:
@dirkmueller this PR is only concerned with creating a virtual ECP cloud Ardana deployment equivalent to the current bare-metal QA deployments. Using 7.5 will require additional work and I'd prefer to have that done in a different PR.

Reply:
@dirkmueller What do you mean by that doesn't exist? you cannot download the image from the link? it works for me.

Regarding using CentOS 7.5, if supporting CentOS 7.5 means we do not need to test CentOS 7.3 anymore, I can work on getting 7.5 to work on this PR instead of 7.3.

Reply:
CentOS 7.3 is getting replaced by CentOS 7.5 within the next few weeks.

Reply:
ok, then I will update this PR with CentOS 7.5 instead of 7.3. Thanks @rsalevsky

Reply:
Sorry, I was mentally replacing centos with rhel in the above link, the link indeed works, but we should be updating that to centos75.